lib/archethic_web/controllers/api/web_hosting_controller/resources.ex

```diff
@@ -0,0 +1,175 @@
+defmodule ArchethicWeb.API.WebHostingController.Resources do
+  @moduledoc false
+
+  use ArchethicWeb, :controller
+
+  alias Archethic
+
+  alias Archethic.TransactionChain.Transaction
+  alias Archethic.TransactionChain.TransactionData
+
+  use Pathex
+
+  require Logger
+  @addresses_key "addresses"
```

Comment:
Same as directory listing, global variables are not usefull I think since they are used only once